src/line_range.rs

```diff
@@ -4,13 +4,15 @@ use crate::error::*;
 pub struct LineRange {
     lower: usize,
     upper: usize,
+    increment: Option<usize>,
```

Comment:
I saw there was a discussion in #1611 about the use case for this feature, but it seems to me like it can be useful.

Maybe you want to let that discussion settle before working on the code more.

Brainstorming: I wonder if the code wouldn't become nicer and more modular if you split up the range in several `LineRange`s, rather than adding an `increment` special case to `LineRange`.

Reply:
Thank you for your comment.
I think it is good to split up the range in several `LineRange`s, too.
I'm considering fixing the code, but I'm not good at rust programming skill.
I'm happy for you to tell me advice or help comment.

Reply:
I changed the code to split up the range in several `LineRange`s.